templates/etc/openldap/default.ldif.erb

```diff
@@ -1,5 +1,5 @@
 dn: <%= @_suffix %>
-dc: <%= @_suffix.split(',').first.split('dc=').last %>
+dc: <%= @_suffix.split(',').first.split('=').last %>
```

Comment:
DN's can contain equal signs, while still not foolproof, we probably should make this `split(/dc=/i)`